lib/render/OSPRay.cpp

```diff
@@ -0,0 +1,200 @@
+#include <vapor/OSPRay.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+using namespace VOSP;
+
+#ifdef BUILD_OSPRAY
+static int _initialized = false;
+
+void ospErrorCallback(OSPError error, const char *msg)
```

Comment:
It looks like the error handler will terminate the application. Can we integrate OSPRay error handling into VAPOR's so that errors are recovered from gracefully? I.e. report the error and continue running.

Reply:
AFAIK this error handler never gets called (in my testing at least, in the case of an error it just crashes 99.9% of the time rather than reporting an error). I just disabled the exit call.